### PR TITLE
chore: update devnet config for EC members

### PR DESCRIPTION
**Pull request by rabi-siddique**
refs: #10194 

## Description
The PR updates the `DEVNET` config in the builder script of replace electorate core eval, which is identical for all testnets. It includes the wallet address of the new EC committee member and adds this address to the high-priority senders list.

### Scaling Considerations
n/a

### Upgrade Considerations
out of scope of this PR; see an upcoming issue

## Security / Documentation / Testing Considerations

These accounts are for testing. The usual key management considerations apply. The relevant private keys are shared on a need-to-know basis for testing.

### CODE REVIEW

File: packages/builders/scripts/inter-protocol/replace-electorate-core.js [hunk 1]
```diff
@@ -55,13 +55,13 @@ const configurations = {
   },
   DEVNET: {
     committeeName: 'Economic Committee',
-    // TODO: Update the addresses after confirmation
     voterAddresses: {
       gov1: 'agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce',
       gov2: 'agoric140dmkrz2e42ergjj7gyvejhzmjzurvqeq82ang',
+      gov3: 'agoric1f0h5zgxyg3euxsqzs0506uj4cmu56y30pqx46s',
```
Comment (dckc) [inline packages/builders/scripts/inter-protocol/replace-electorate-core.js]: That's gov4, isn't it?

File: packages/builders/scripts/inter-protocol/replace-electorate-core.js [hunk 2]
```diff
@@ -55,13 +55,13 @@ const configurations = {
...
     },
     highPrioritySendersConfig: {
-      addressesToAdd: [],
+      addressesToAdd: ['agoric1f0h5zgxyg3euxsqzs0506uj4cmu56y30pqx46s'],
       addressesToRemove: ['agoric1w8wktaur4zf8qmmtn3n7x3r0jhsjkjntcm3u6h'],
```
Comment (dckc) [inline packages/builders/scripts/inter-protocol/replace-electorate-core.js:65]: how do we remove `...u6h`? I don't see it listed in `voterAddresses`.

**Review by dckc — COMMENTED** [review]
I need help reading this

File: packages/builders/scripts/inter-protocol/replace-electorate-core.js (hunk 1 above)
Comment (rabi-siddique) [reply packages/builders/scripts/inter-protocol/replace-electorate-core.js]: Yes, `gov3` is actually `gov4` in my sequential numbering. Sorry for any confusion! Do you suggest naming it `gov4`?

File: packages/builders/scripts/inter-protocol/replace-electorate-core.js (hunk 2 above)
Comment (rabi-siddique) [reply packages/builders/scripts/inter-protocol/replace-electorate-core.js:65]: The `voterAddresses` map includes the addresses that will be invited to join the new committee and charter. The lists `addressesToAdd` and `addressesToRemove` are linked to `highPrioritySenders`. The new member `...x46s`, has been added to `addressesToAdd`. An outgoing member from the previous committee, .`..u6h`, has been placed in `addressesToRemove`.

**Comment by dckc** [comment]
Again, this can't be in service to #10209 ; that issue is for _post code complete testing_; that is: stuff that happens after all git commits.

I think the relevant issue here is:
 - #10194

**Comment by rabi-siddique** [comment]
> Again, this can't be in service to #10209 ; that issue is for _post code complete testing_; that is: stuff that happens after all git commits.
> 
> I think the relevant issue here is:
> 
> * [Update hardcoded EC addresses with newly elected committee members #10194](https://github.com/Agoric/agoric-sdk/issues/10194)

@dckc Made the update in PR description.

File: packages/builders/scripts/inter-protocol/replace-electorate-core.js (hunk 1 above)
Comment (dckc) [reply packages/builders/scripts/inter-protocol/replace-electorate-core.js]: yes, please!

File: packages/builders/scripts/inter-protocol/replace-electorate-core.js (hunk 2 above)
Comment (dckc) [reply packages/builders/scripts/inter-protocol/replace-electorate-core.js:65]: ok... I recognize `...u6h` as gov3 now, and I see it's in the [devnet highPrioritySenders](https://vstorage.agoric.net/?path=highPrioritySenders&endpoint=https%3A%2F%2Fdevnet.rpc.agoric.net%3A443&height=null).

File: packages/builders/scripts/inter-protocol/replace-electorate-core.js (hunk 1 above)
Comment (rabi-siddique) [reply packages/builders/scripts/inter-protocol/replace-electorate-core.js]: Done

**Comment by dckc** [comment]
**Upgrade considerations** are actually non-trivial. I think we're running this core-eval as part of a chain-halting upgrade, so we need to make sure it's hooked into the go upgrade handler. IOU an issue, I suppose. I suggest changing "n/a" to "out of scope of this PR; see an upcoming issue".

I suggest combining **Security Considerations**, **Documentation Considerations** and **Testing Considerations** into one section:

## Security / Documentation / Testing Considerations

These accounts are for testing. The usual key management considerations apply. The relevant private keys are shared on a need-to-know basis for testing.

**Review by dckc — APPROVED** [review]
looks good, presuming an update to the PR description

**Pull request merged**